packages/stackable/stackables/fastify.js

```diff
@@ -35,4 +36,20 @@ class FastifyStackable extends BaseStackable {
   }
 }
 
-module.exports = FastifyStackable
+async function buildFastifyStackable (options) {
+  // TODO: load the config if options is a string
+
+  const configManager = options.configManager
+  const config = configManager.current
+
+  const entrypointPath = config.entrypoint || 'index.js'
+  const entrypointFullPath = join(options.configManager.dirname, entrypointPath)
+
+  const { app } = require(entrypointFullPath)
+  return new FastifyStackable({ app, config, configManager })
+}
+
+module.exports = {
+  FastifyStackable,
+  build: buildFastifyStackable,
+}
```

Comment:
This should match (or potentially be a subset of) https://github.com/platformatic/platformatic/pull/2887/files#diff-ba257106a2aba7ce5a1349dc2e144300c707a6449da9be8ab6a6cdaf1a06f810R72.